pkgs/top-level/all-packages.nix

```diff
@@ -12227,7 +12227,7 @@ with pkgs;
   qemu_xen = lowPrio (qemu.override { x86Only = true; xenSupport = true; xen = xen-slim; });
   qemu_xen-light = lowPrio (qemu.override { x86Only = true; xenSupport = true; xen = xen-light; });
 
-  qemu_test = lowPrio (qemu.override { x86Only = true; nixosTestRunner = true; });
+  qemu_test = lowPrio (qemu.override { x86Only = false; nixosTestRunner = true; });
```

Comment:
This might depend on my changes to pkgs/applications/virtualization/qemu/default.nix ... thoughts?